src/storage/src/sink/kafka.rs

```diff
@@ -832,18 +831,7 @@ impl KafkaSinkState {
 
         let mut progress_emitted = false;
 
-        // This only looks at the first entry of the antichain.
-        // If we ever have multi-dimensional time, this is not correct
-        // anymore. There might not even be progress in the first dimension.
-        // We panic, so that future developers introducing multi-dimensional
-        // time in Materialize will notice.
```

Comment:
maybe you should keep the comment and re-word it slightly

Reply:
I removed the comment too on the basis its content is saying something about how to correctly use `Antichains` and not something about this particular use, which is why the other sites are not commented either. My expectation is that if an engineer in the future wants to introduce partially ordered time here they will know what they are doing and understand why this can't compile anymore.